frontend/packages/component-library/src/cms/tabGroup/tabGroup.module.scss

```diff
@@ -45,10 +45,10 @@ $mobileBreakpoint: 1024px;
   background: var(--background-neutral-primary);
 }
 
-.tabGroupModelTabImageContainer {
+figure.tabGroupModelTabImageContainer {
   width: 28.6699rem;
-
-  border-radius: variables.$regular-border-radius;
+  height: 18.7234rem;
```

Comment:
Is this a magic number or what we always want this to be? (Same question for line 92)

Reply:
+1 could you explain the value?

Reply:
That's just values from Figma. Image will always have fixed width and height based on Figma :)

Reply:
Ok cool, that makes sense! I assume we'll do the same for ActionBlock too?

Reply:
Could you add a comment explaining this and where to get the numbers from should they update?

Reply:
> Could you add a comment explaining this and where to get the numbers from should they update?

I think that's redundant, as it is practically the same as if we would write a comment for any other line of styles as it's the same thing

Reply:
> Ok cool, that makes sense! I assume we'll do the same for ActionBlock too?

Absolutely, if image has fixed dimensions in figma, we should set them to the component as well

Reply:
If there's not much visual difference could we trim the decimals and just round these to like `18.7` instead?

Reply:
We have a linter rule for that and this value is already shortened to fit it. Trim more decimals may give unexpected regressions, so I'll just add a comment about where these values are taken from and why as you suggested above.

But in a follow-up PR. :) I will merge this PR now as is not to block anyone

Reply:
Sounds good, ty! I made a [ticket](https://codedotorg.atlassian.net/browse/CMS-491) to update this for the ActionBlocks too I can do this sprint 🙌